flake.nix

```diff
@@ -131,5 +136,29 @@
         doCheck = false;
         cargoSha256 = "sha256-A24O3p85mCRVZfDyyjQcQosj/4COGNnqiQK2a7nCP6I=";
       };
+
+      packages.default = rustPlatform.buildRustPackage {
+        name = "fission-server";
+        src = ./.;
+        cargoLock = {
+          lockFile = ./Cargo.lock;
+          outputHashes = {
+            "rs-ucan-0.1.0" = "sha256-bJq5mMk5TLvE9w1dtY1sRCyE2uVfcMFSupsSFTXw9rY=";
+          };
+        };
+        buildInputs = with pkgs;
+          [openssl postgresql rust-toolchain]
+          ++ lib.optionals stdenv.isDarwin [
+            darwin.apple_sdk.frameworks.Security
+            darwin.apple_sdk.frameworks.CoreFoundation
+            darwin.apple_sdk.frameworks.Foundation
+          ];
+
+        doCheck = false;
+
+        nativeBuildInputs = with pkgs; [pkg-config];
+
+        OPENSSL_NO_VENDOR = 1; # see https://github.com/sfackler/rust-openssl/pull/2122
+      };
```

Comment:
Oh this looks so easy.

I'll need to remember to update that hash though.
Updating hashes was a matter of running it with the wrong hash, getting an error and just sticking the new hash from the error in there?

Reply:
Yeah exactly — or at least that's what I do

Reply:
Yeah... and the hash is *only* needed for rs-ucan since we're not using a crates.io release, once that changes this should just work ™️ (see the homestar version).

Reply:
Yupyup alright